### PR TITLE
[14.0][ADD] fleet_vehicle_capacity: add new module

### PR DESCRIPTION
**Pull request by marcelsavegnago**
Implementing a new module to define vehicle capacities, such as weight, volume, and passengers. This module should be flexible enough to allow the addition of more fields of this type in the future, if necessary.

Regarding the vehicle, we already have the 'seats' field. Is it really necessary to implement a separate field for the number of passengers it can accommodate?

cc @max3903 @pedrobaeza

### CODE REVIEW

File: fleet_vehicle_capacity/models/fleet_vehicle.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        string="Volume Capacity",
+        help="The maximum volume capacity the vehicle can support.",
+        readonly=False,
+    )
+
+    passenger_capacity = fields.Float(
+        related="model_id.passenger_capacity",
+        string="Passenger Capacity",
+        help="The maximum passenger capacity the vehicle can support.",
+        readonly=False,
```
Comment (ivantodorovich) [inline fleet_vehicle_capacity/models/fleet_vehicle.py]: It feels prone to error to have `readonly=False` here.

Let's say I have a fleet with many vehicles, they're all the same model.
For one of the vehicles, I've a modified/disabled seat, so I'm tempted to decrease this passenger capacity.
I'll be unknowingly modifying the model, and all the other vehicles too, wouldn't I?

I'd then go for any of the following two alternatives:

1) Have these fields be computed from the model with `@api.depends('model_id')` (achieving the onchange+default effect). Then, users can modify these values on a per vehicle basis.

2) Have these related fields as readonly, and so it encourage users to modify the model instead

**Review by ivantodorovich — APPROVED** [review]

**Review by WesleyOliveira98 — APPROVED** [review]
LGTM

File: fleet_vehicle_capacity/models/fleet_vehicle.py (hunk 1 above)
Comment (marcelsavegnago) [reply fleet_vehicle_capacity/models/fleet_vehicle.py]: @ivantodorovich.

Done. I chose to restrict the modification of these parameters on the vehicle itself, forcing any changes to be defined at the vehicle model level.

**Pull request merged**